scalameta/common/shared/src/main/scala/scala/meta/internal/trees/ast.scala

```diff
@@ -722,15 +719,14 @@ class AstNamerMacros(val c: Context) extends Reflection with CommonNamerMacros {
 
 object AstNamerMacros {
 
-  private type Version = (Int, Int, Int)
-  private val versionOrdering = implicitly[Ordering[Version]]
+  type Version = (Int, Int, Int)
+  val versionOrdering = implicitly[Ordering[Version]]
 
   private def versionToString(v: Version): String =
     s"${v._1}.${v._2}.${v._3}"
 
-  private def parseVersion(v: String): Version = {
-    val since = v.stripPrefix("\"").stripSuffix("\"")
-    val versions = since.split('.').map(_.toInt)
+  def parseVersion(v: String, delim: Char = '.'): Try[Version] = Try {
```

Comment:
Actually, why do we need to make it public? Could we move it outside of  AstNamerMacros to some kind of Version class/object?

Reply:
done, thank you